apps/api/src/app/benchmark/benchmark.service.ts

```diff
@@ -217,6 +224,17 @@ export class BenchmarkService {
   } & UniqueAsset): Promise<BenchmarkMarketDataDetails> {
     const marketData: { date: string; value: number }[] = [];
 
+    const endDate = new Date();
```

Comment:
Can you extend the function arguments by `endDate = new Date();` (`endDate?: Date;`) instead of this?

Reply:
Done :+1: